source/plugin/ray-tracing.rst

```diff
@@ -61,7 +60,14 @@ Finally, use :javadoc:`BlockRay.BlockRayBuilder#build()` to finish building the
 
     Player player;
     BlockRay<World> blockRay = BlockRay.from(player)
-        .filter(BlockRay.continueAfterFilter(BlockRay.onlyAirFilter(), 1)).build();
+        .stopFilter(BlockRay.continueAfterFilter(BlockRay.onlyAirFilter(), 1)).build();
+
+We can rewrite the above to use skipFilter in addition to stopFilter. This will skip all air blocks and stop at the first non-air block it hits.
```

Comment:
\`\`BlockRay#skipFilter(Predicate<BlockRayHit<E>>)\`\`, \`\`BlockRay#stopFilter(Predicate<BlockRayHit<E>>)\`\` instead of skipFilter and stopFilter. Alternatively, make these javadoc links.